frontend/src/app/privacy-policy/page.tsx

```
@@ -0,0 +1,199 @@
+'use client';
+
+import React from 'react';
+import CustomMarkdown from 'src/components/CustomMarkdown';
+
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)